doc/flatpak-run.xml

```diff
@@ -752,6 +820,9 @@ key=v1;v2;
         <para>
             <command>$ flatpak run --command=bash org.gnome.Sdk</command>
         </para>
+        <para>
+            <command>$ flatpak run org.gnome.Boxes --no-usb=0fd9:*</command>
```

Comment:
There is an issue using `*` here is that on most shell the expression is expanded.

Here on zsh:
```
% flatpak run org.gnome.Boxes --nousb=0fd9:*
zsh: no matches found: --nousb=0fd9:*
```

This works though:
```suggestion
            <command>$ flatpak run org.gnome.Boxes --no-usb=0fd9:\*</command>
```

Reply:
Maybe instead the syntax could be `VEND:` for "all devices from vendor VEND"